change/@ni-nimble-blazor-f8fbcdc6-38cc-4c51-8016-115c43182376.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@munteannatan can you give an update on status of the PR and expectations?
As an fyi most reviewers will be OOO W-F next week.

Reply:
Sorry about this, we are also OOO until next week